tests/append_test_data.py

```diff
@@ -0,0 +1,131 @@
+"""
+Use this script to append data from the test.yml file to a sample.
+
+Dictionary format:
+{
+ "fva_string (0x%x)":["string1","string2",...]
+ <...>
+ optional: "all":["stringN",...]
+}
+
+Example:
+{
+ "0x401000":["hostid","SYSTEM"]
+ "all":["8.8.8.8","explorer.exe"]
+}
+"""""
+
+import os
+import sys
+import yaml
+import json
+import struct
+from pprint import pprint
+
+
+FILE_START = 0
+FILE_END = 2
+
+MAGIC = "FLSS"
+SIZE_OFFSET = 4
+SIZE_LEN = 4
+SIZE_MAGIC = len(MAGIC)
+
+
+def contains_magic(sample_path):
```

Comment:
minor style nit: if a function returns a boolean, recommend naming it starting with `is_`, `has_`. or `does_`.